constants.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
is there any way for the code to make the json file have 4 spaces per tab? currently reading the json is pretty hard.